recipes/intbitset/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "intbitset" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d4ceba6c252223a2e60072b8e42b867004eb31972098e0d16924cf0d27aed387
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - six
```

Comment:
Is `six` needed for build? Looks like not.

Reply:
I think six is needed so I've added it back in

Reply:
Only remove it in `host`, not in `run`. It is required to run the package but not to build it.